{"name": "test_login[test_data4]", "status": "passed", "attachments": [{"name": "log", "source": "f58a99cf-ef07-4478-b41a-1beaf87c76ee-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "0c4c1272-d38e-4508-8b9c-8a2b1dceaf5e-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "test_data", "value": "{'case_id': 5, 'interface': 'login', 'title': '错误用户名', 'method': 'post', 'url': '/api/OAuth/Token', 'data': '{\\n  \"grant_type\": \"password\",\\n  \"username\": \"testtest\",\\n  \"password\": \"123456\",\\n  \"uuid\": 690578652,\\n}', 'expected': \"{'code': -1, 'msg': '用户名错误'}\", 'sql': None, 'result': '通过'}"}], "start": 1613790459866, "stop": 1613790460191, "uuid": "760c72dd-d53d-4553-b411-ddaf67160503", "historyId": "dbb92fe366a08e5391877711bcc252ac", "testCaseId": "2116f0ad00ab1de0effc54f114dd9496", "fullName": "tests.test_login.TestLogin#test_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-8M7P7VT"}, {"name": "thread", "value": "15180-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}